handlers/user.go

```diff
@@ -6,12 +6,13 @@ import (
 	"net/http"
 	"strconv"
 
+	"github.com/gorilla/mux"
 	"github.com/somos831/somos-backend/models"
 	"github.com/somos831/somos-backend/responses"
 )
 
-const (
-	UserNotFoundErr = "user not found"
+var (
+	UserNotFoundErr = errors.New("user not found")
```

Comment:
I think for this project it's okay to have global variables. We just need to be extra careful that we are not sharing mutable state across different files and packages. 

Could you edit the `.golangci.yml` config to remove the `gochecknoglobals` linter?

Reply:
@villaleo I updated the linter settings now, it should be good to go.